go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -284,6 +285,26 @@ var metricBANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of baseline admin network policies (0 or 1) in the cluster",
 })
 
+var metricANPDBObjects = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
@npinaeva and @martinkennelly I have tried one cardinality based on what we spoke, this looked like better view for user, so each feature == 1 metric and then table_Name is the cardinality.
versus do we actually want totally just 1 metric with featureName/owner also becoming a cardinality? I don't personally want to do 2D metric to avoid complexity  but wdybt?

Reply:
In terms of cardinality and Prometheus load, there's no difference between
* 2 metrics with a cardinality of 10 each = 20 series.
* 1 metric with a cardinality of 20 = 20 series.

The key question is: do you expect to run aggregations on the 2 metrics at the same time? E.g. if "sum(admin_network_policies{...}) + sum(baseline_admin_network_policies{...})" is a meaningful query then you want 1 metric with a label differentiating between admin/baseline_admin.

Reply:
> In terms of cardinality and Prometheus load, there's no difference between
> 
> * 2 metrics with a cardinality of 10 each = 20 series.
> * 1 metric with a cardinality of 20 = 20 series.

thanks Simon! that deff clears up the complexity wise how that is interpreted in prom

> 
> The key question is: do you expect to run aggregations on the 2 metrics at the same time? 

hmm that deff makes it simpler for us to think about.. I think we are ok to treat these features differently and go with per CRD metrics. 

> E.g. if "sum(admin_network_policies{...}) + sum(baseline_admin_network_policies{...})" is a meaningful query then you want 1 metric with a label differentiating between admin/baseline_admin.